docker/third-party/images.txt

```diff
@@ -12,8 +12,10 @@ python:3.10
 python:3.10-slim
 redis:6.0.6-alpine
 ubuntu:18.04
-ubuntu:19.04
+ubuntu:20.04
```

Comment:
Is this a deliberate edit (instead of addition)?

Reply:
Yes. I could leave it or delete it. I'm not sure we seriously used 20.04 at any point, but 19.04 is certainly out of support at this time.

Reply:
Furthermore, we had a pinned `focal` version, which is 20.04. So I think this may have just been a bug.